styles.css

```diff
@@ -0,0 +1,435 @@
+
+.editors-pick {
+  width: 240px;
+  float: right;
+  margin-right: 50px;
+  top: 20px;
+}
+
+.editors-pick h6 {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-weight: bold;
+  margin: 0;
+}
+
+.editors-pick img {
+  float: left;
+  margin-right: 10px;
+}
+
+.editors-pick p {
+  font-size: 14px;
+  line-height: 120%;
+  padding-bottom: 10px;
+}
+
+.editors-pick a {
+  color: #333;
+}
+
+/* ARTICLE PROPERTIES ======================================================== */
+
+/* Article Title */
+
+.article h1 {
+  line-height: 120%;
+  font-style: italic;
+  font-size: 36px;
+  color: black;
+}
+
+.sharing-btn {
+  border: 0;
+  border-radius: 50%;
+  background-color: #ececec;
+  padding: 8px;
+  scale: 0.7;
+  height: 40px;
+}
+
+/* Article Text */
+
+.article {
+  color: #333;
+  line-height: 130%;
+  width: 30%;
+  margin: 0 auto;
+}
+
+/* IMAGE FROM ARTICLE PROPERTIES ============================================= */
+
+.article-img {
+  margin: 0 auto;
+}
+
+.article-main-img {
+  width: 63%;
+}
+
+.article-explanation-img {
+  width: 940px;
+  position: relative;
+}
+
+.article-explanation-img img {
+  z-index: -1;
+}
+
+.row-text-box {
+  position: absolute;
+  display: flex;
+}
+
+.top-4 {
+  bottom: 370px;
+}
+
+.bottom-4 {
+  bottom: 80px;
+}
+
+.single-text-box {
+  width: 220px;
+  margin-right: 20px;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 12px;
+}
+
+.article-final-img {
+  width: 40%;
+}
+
+.article-img h4,
+.article-img h6 {
+  margin: 0;
+}
+
+/* Captions */
+
+.caption {
+  font-size: 17px;
+  color: #666;
+  line-height: 110%;
+  margin: 0;
+}
+
+.caption small {
+  color: #888;
+}
+
+.article-figcaption {
+  width: 85%;
+}
+
+.article-videocaption {
+  width: 90%;
+}
+
+.article-details {
+  font-size: 13px;
+}
+
+/* BUTTONS PROPERTIES ======================================================= */
+
+.read-btn {
+  font-size: 13px;
+  padding: 10px 80px;
+}
+
+.final-share {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+/* OTHER NEWS PROPERTIES ==================================================== */
+
+.suggestions {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  width: 60%;
+  margin: 0 auto;
+}
+
+/* Detailed Content Articles */
+
+.detailed-content {
+  margin-right: 3%;
+  max-width: 70%;
+}
+
+.card-row {
+  display: flex;
+}
+
+.card {
+  border: 1px solid #e2e2e2;
+  transition-duration: 0.3s;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  padding: 15px;
+  position: relative;
+}
+
+.news-extra {
+  font-size: 12px;
+  display: block;
+  margin-top: 4px;
+  color: #666;
+}
+
+.align-right {
+  text-align: right;
+}
+
+.news-title {
+  font-size: 15px;
+  margin: 0 0 4px 0;
+}
+
+.card-margin {
+  margin-right: 2%;
+}
+
+.card:hover {
+  box-shadow: 0 0 8px #e2e2e2;
+  cursor: pointer;
+}
+
+.card p {
+  font-size: 14px;
```

Comment:
Always try to avoid the repetition of the same rule(s) as much as you can. You should please DRY your code in the above blocks. 
![image](https://user-images.githubusercontent.com/65501535/108138438-09ce7a80-70cf-11eb-8ad3-05a944bfd24f.png)

`font-size: 14px` has a number of occurences.